### PR TITLE
Add pybind param names to docstring for 'TSDFVoxelGrid.integrate'

### PR DESCRIPTION
**Pull request by Algomorph**
This change is [Reviewable](https://reviewable.io/reviews/intel-isl/open3d/3058)

### CODE REVIEW

**Comment by Algomorph** [comment]
Running the style-check now, sry

**Comment by Algomorph** [comment]
Ok, not sure why the documentation build is failing (see logs). Could it be that I'm trying to PR from `master` on my fork instead of an alternative branch name? If so, is that an issue that should be addressed on my end?

**Comment by yxlao** [comment]
thanks @Algomorph , the documentation build issue seems to be a glitch related to network

File: cpp/pybind/t/geometry/tsdf_voxelgrid.cpp [hunk 1]
```diff
@@ -56,12 +56,17 @@ void pybind_tsdf_voxelgrid(py::module& m) {
     tsdf_voxelgrid.def("integrate",
                        py::overload_cast<const Image&, const core::Tensor&,
                                          const core::Tensor&, float, float>(
-                               &TSDFVoxelGrid::Integrate));
+                               &TSDFVoxelGrid::Integrate),
+                       "depth_image"_a, "intrinsics"_a, "extrinsics"_a,
```
Comment (yxlao) [inline cpp/pybind/t/geometry/tsdf_voxelgrid.cpp:60]: Let's be consistent with the C++ name: `depth_image` -> `depth`.

File: cpp/pybind/t/geometry/tsdf_voxelgrid.cpp [hunk 2]
```diff
@@ -56,12 +56,17 @@ void pybind_tsdf_voxelgrid(py::module& m) {
...
+                       "depth_scale"_a, "depth_max"_a);
+
     tsdf_voxelgrid.def(
             "integrate",
             py::overload_cast<const Image&, const Image&, const core::Tensor&,
                               const core::Tensor&, float, float>(
-                    &TSDFVoxelGrid::Integrate));
+                    &TSDFVoxelGrid::Integrate),
+            "depth_image"_a, "color_image"_a, "intrinsics"_a, "extrinsics"_a,
```
Comment (yxlao) [inline cpp/pybind/t/geometry/tsdf_voxelgrid.cpp:68]: Let's be consistent with the C++ name: `depth_image` -> `depth`, `color_image` -> `color`.

**Review by yxlao — CHANGES_REQUESTED** [review]

File: cpp/pybind/t/geometry/tsdf_voxelgrid.cpp (hunk 1 above)
Comment (Algomorph) [reply cpp/pybind/t/geometry/tsdf_voxelgrid.cpp]: Done!

File: cpp/pybind/t/geometry/tsdf_voxelgrid.cpp (hunk 2 above)
Comment (Algomorph) [reply cpp/pybind/t/geometry/tsdf_voxelgrid.cpp]: Done!

**Review by yxlao — APPROVED** [review]
Thanks.

Will merge after CI checks.

**Pull request merged**